src/redis-benchmark.c

```diff
@@ -1495,6 +1496,11 @@ int parseOptions(int argc, char **argv) {
             fprintf(stderr,
                     "WARNING: -e option has no effect. "
                     "We now immediately exit on error to avoid false results.\n");
+        } else if (!strcmp(argv[i],"--seed")) {
+            if (lastarg) goto invalid;
+            config.rand_seed = atoi(argv[++i]);
+            srandom(config.rand_seed);
+            init_genrand64(config.rand_seed);
```

Comment:
It seems this is the only place `rand_seed` is used, so there's no need to save it in config, is it?

Reply:
> Maybe mention 'redis-benchmark' in the PR title?

Updated title.



> It seems this is the only place `rand_seed` is used, so there's no need to save it in config, is it?

Yes, I agree. I had that thought when I was writing it as well. I had done this in the first place so thought I'd leave it in in case it's needed to be referenced anywhere else in the code. Realistically, there's probably no other use aside from maybe printing it out.